config/lists/events.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" ?>
+<list xmlns="http://schemas.sulu.io/list-builder/list">
+    <key>events</key>
+
+    <properties>
+        <property name="id" visibility="no" translation="sulu_admin.id">
+            <field-name>id</field-name>
+            <entity-name>App\Entity\Event</entity-name>
+        </property>
+
+        <property name="name" visibility="always" searchability="yes" translation="sulu_admin.name">
+            <field-name>name</field-name>
+            <entity-name>App\Entity\Event</entity-name>
+        </property>
+
+        <property name="startDate" visibility="yes" translation="app.start_date" type="date">
```

Comment:
Available values for the `type` attribute are printed to the development console of the browser when an invalid type such as `invalid-property-type` is used in a list configuration.